docs/Developers/Writing_Native_Bears.rst

```diff
@@ -531,13 +531,13 @@ can be used as follows:
 Aspect Bear
 -----------
 
-Aspect is a feature in coala that make configuring coala in project more easy
+Aspect is a feature in coala that makes configuring coala in project more easy
 and language agnostic. For more detail about aspect, see cEP-0005 in
 https://github.com/coala/cEPs/blob/master/cEP-0005.md.
 
 An aspect-compliant bear MUST:
 
-1. Declare list of aspect it can fix and detected. Note that the aspect MUST be
+1. Declare list of aspect it can detect and fix. Note that the aspect MUST be
    a leaf aspect. You can see list of supported aspect here
```

Comment:
You could add 'the here':  `You can see the list of...`

Reply:
I think `must` should in lower case letters @AprajitaSingh